storage/store.go

```diff
@@ -1476,7 +1476,9 @@ func (s *Store) processRaft() {
 						if err != nil {
 							log.Fatal(err)
 						}
-						log.Infof("got an event membership change committed event: %s", cmd.Cmd)
+						if log.V(6) {
+							log.Infof("store %s: new committed membership change at inded %d", s, e.Index)
```

Comment:
actually exactly this line is very helpful in debugging #2593 which just resurfaced. Would you mind leaving it in place for now?